src/parseTorrent.ts

```diff
@@ -112,6 +186,21 @@ export class Metafile {
 			this.isSingleFileTorrent = false;
 		}
 		this.title = parseTitle(this.name, this.files) ?? this.name;
+		this.tags = [];
+		this.trackers =
+			raw["announce-list"]?.map((tier) =>
+				tier
+					.map((url) => sanitizeTrackerUrl(url.toString()))
+					.filter(isOk)
+					.map((r) => r.unwrap()),
+			) ??
```

Comment:
there's probably a pretty easy way to share this code across `announce` and `announce-list` (and I just saw a third one in `updateMetafileMetadata`).